motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 <xacro:macro name="motoman_motopos_d500" params="prefix">
-
+  <xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
+  <xacro:property name="specific_robot_config" value="${config.specific_robot_config}"/>
```

Comment:
```suggestion
```
Remove this. The robot config is a forge generic robot construct, this should be in there if it is needed.